bcp/client/src/webpack/pages/List.js

```diff
@@ -1,13 +1,47 @@
-import React from 'react';
-
-
-const List = () => {
+import React, {useState, useEffect} from 'react';
+import useStores from "../hooks/use-stores";
+import {observer} from "mobx-react";
+import Table from 'react-bootstrap/Table';
+import {ASSET_MODE} from "../constant";
+import {fetchAircraftGeoJSON, fetchEquipmentGeoJSON} from "../helper/toGeoJSON";
+import {getAssetCell} from '../helper/map-dashboard';
 
+const List =observer(() => {
+    const {mapStore} = useStores();
+    const assetMode = mapStore.assetType;
+    const [dataSource, setDataSource] = useState(null);
+    useEffect(()=>{
+        if(assetMode===ASSET_MODE.AIRCRAFT){
+            fetchAircraftGeoJSON(setDataSource);
+        }else{
+            fetchEquipmentGeoJSON(setDataSource);
+        }
+    },[]);
     return (
         <div>
-            List
+            <Table striped>
+                <thead>
+                <tr>
+                    <th>Asset</th>
+                    <th>Base Location</th>
+                    <th>Status</th>
+                    <th>F-CTAF</th>
+                    <th></th>
+                </tr>
+                </thead>
+                <tbody>
+                {dataSource.features.map(feature=>{
```

Comment:
Surely this will produce an exception as dataSource will be null on first render.

Reply:
will update this on ABCP-11 for list page